website/addons/base/views.py

```diff
@@ -202,7 +204,17 @@ def create_waterbutler_log(payload, **kwargs):
     auth = Auth(user=user)
     node = kwargs['node'] or kwargs['project']
 
-    if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED):
+    try:
+        dest = payload['destination']
+        src = payload['source']
+        if src is not None and dest is not None:
+            if os.path.split(dest['materialized'])[0] == os.path.split(src['materialized'])[0]:
```

Comment:
First, shouldn't the name in source and destination be different the it is a rename action, which means this line should be 

if os.path.split(dest['materialized'])[0] != os.path.split(src['materialized'])[0]:

 Second, if you have an two folders A and B within an addon, move a subfolder from A to B will make the path to be different as source one will be /A/subfolder and destination will be /B/subfolder, but clearly this is not a renaming action but a move action. In this case, you can change this line to:
 if os.path.split(dest['materialized'])[0].split('/')[-1] != os.path.split(src['materialized'])[0].split('/')[-1] :

Finally we come to the extreme fun case:
if under one addon user has a folder named A which has a subfolder named C, and under the same addon user has a folder named B which also has a subfolder named C with different content from A. Then user move subfolder C under folder A into folder B, it will pop up an option ask user what you want to do with the two folder has the same name, you can choose to keep both, then the system will auto changed the name of the folder you moved to C(1). However your log will show the user behavior is rename not move.